mibios/glamr/templates/glamr/sample_detail.html

```diff
@@ -0,0 +1,36 @@
+{% extends "glamr/base.html" %}
+{% block content %}
+
+<nav aria-label="breadcrumb">
+  <ol class="breadcrumb">
+    <li class="breadcrumb-item"><a href="/">Home</a></li>
+    <li class="breadcrumb-item"><a href="/{{ detail_url }}">Data Set</a></li>
```

Comment:
I did quite follow the changes to the SampleView class, does your get_detail_url() do anything that the url tag suggested below doesn't?
```suggestion
    <li class="breadcrumb-item"><a href="{% url 'dataset' pk=object.dataset.pk %}">Data Set</a></li>
```

Reply:
It does not- thanks for the suggestion, I didn't know how to do this! I've removed get_detail_url and now do this instead